sourcemash/frontend/frontend.py

```diff
@@ -1,9 +1,13 @@
 from . import bp
 
-from flask import render_template
+from flask import render_template, url_for
 
 
 @bp.route('/')
 @bp.route('/index')
 def index():
     return render_template('index.html')
+
+@bp.route('/login')
```

Comment:
We do catch the click.

However, submit on this login form builds the URL ```.../login``` with variables, which we want to redirect back to index. We want this: (1) so it doesn't show the variables, and (2) because we don't want the login_user.html default template.